plugin-modernizer-core/src/main/java/io/jenkins/tools/pluginmodernizer/core/impl/MavenInvoker.java

```
@@ -150,6 +150,7 @@ private void invokeGoals(Plugin plugin, String... goals) {
             handleInvocationResult(plugin, result);
         } catch (MavenInvocationException e) {
             LOG.error(plugin.getMarker(), "Maven invocation failed: ", e);
+            plugin.addError(e);
```

Was silently failing :(